source/install/cloud-changelog.md

```diff
@@ -4,12 +4,43 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-04-13](#release-2023-04-13)
 - [Release 2023-03-29](#release-2023-03-29)
 - [Release 2023-03-20](#release-2023-03-20)
 - [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-16](#release-2023-01-16)
 - [Release 2022-12-20](#release-2022-12-20)
-- [Release 2022-12-01](#release-2022-12-01)
+
+## Release 2023-04-13
+
+### Improvements
+
+#### User Interface (UI)
+ - Enabled Work Templates in Onboarding checklist and `+` Menu.
+ - Added a marketplace button to the bottom of the apps bar. The button is visible if the marketplace is enabled and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.
+ - Added a "Add channels" button to the bottom of the left-hand sidebar to make it more obvious for users on how to create or join channels.
+ - Removed Webapp Build Hash since it is now identical to Server Build Hash.
+
+#### Administration
+ - The following repositories have been merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. Playbooks and Boards are now core parts of the product that cannot be disabled.
+ - The file info stats query is now optimized by denormalizing the ``channelID`` column into the table itself. This will speed up the query to get the file count for a channel on clicking the right-hand side. Migration times:
```

Comment:
```suggestion
 - The file info stats query is now optimized by denormalizing the ``channelID`` column into the table itself. This will speed up the query to get the file count for a channel when selecting the right-hand pane. Migration times:
```

Reply:
Engineering: Please confirm that my editorial suggestions haven't changed the intended meaning of this bullet point. Thanks!